tests/tls_e2e/host/host.cpp

```diff
@@ -331,7 +331,8 @@ int run_scenarios_tests()
                 "This is a %s test case. Expect %s errors\n",
                 (unittests_configs[j].negative_test ? "negative" : "positive"),
                 (unittests_configs[j].negative_test ? "" : "no"));
-
+            if (j == 1)
+                sleep(90);
             ret = run_test_with_config(&test_configs);
```

Comment:
This delay is introduced intentionally. After successful OpenSSL client/server communication, sockets are staying in TIME_WAIT state for a minute. [link ](https://superuser.com/questions/173535/what-are-close-wait-and-time-wait-states ). Issue is observed on ubuntu too. I am investigating this to find a better way to resolve this issue. I added this intermediate fix so that code can be reviewed. I will fix this before merging to master.

Reply:
Can you add comments about this issue?

Reply:
Added a new issue to track this https://github.com/openenclave/openenclave/issues/3738. I still could not find a solution for it and not sure if there exists one. I will pick it up after completing rest of release activities as this is not a major blocker. @radhikaj